mica-webapp/src/main/webapp/app/entity-taxonomy-config/views/entity-taxonomy-config-properties-view.html

```diff
@@ -5,22 +5,25 @@ <h3 ng-if="model.type==='taxonomy'">
       <li class="active">{{model.content.name}}</li>
     </ol>
   </h3>
-  <h3 ng-if="model.type!=='taxonomy'">
-    <ol class="breadcrumb mica-breadcrumb">
-      <li><a href ng-click="navigateToTaxonomy()">{{taxonomy.name}}</a></li>
-      <li class="active">
-        <span>{{model.content.name}}</span>
-        <span ng-if="target.editable && !isStatic(model.content)">
-              <a href ng-click="deleteCriterion()"><i class="fa fa-trash"></i></a>
-            </span>
-      </li>
-    </ol>
-  </h3>
 
-  <h4>
-    <span translate>properties</span>
+  <div class="row" ng-if="model.type!=='taxonomy'">
+    <div class="col-md-9">
```

Comment:
if content is smaller than col-md-9, the icon won't stick to the title.

Reply:
I am not using the icon, I am using a button to the right.

Reply:
Amended fix.